src/pages/PositionsList/index.tsx

```diff
@@ -48,11 +48,12 @@ export const PositionsList = () => {
     {
       // eslint-disable-next-line react/display-name
       cell: (row: Position) => {
-        return networkConfig ? (
-          <TokenIcon symbol={networkConfig.getTokenFromAddress(row.collateralToken).symbol} />
-        ) : (
-          row.collateralToken
-        )
+        try {
+          const token = networkConfig && networkConfig.getTokenFromAddress(row.collateralToken)
+          return <TokenIcon symbol={token.symbol} />
+        } catch {
```

Comment:
Good work, What about to add the `(error)`  parameter and display it in the console with `logger.error(error)` ? What do you think ?

Reply:
👍